### ISSUE TITLE
Typo in apps_deployment.go

### ISSUE THREAD

**Issue by gitfool** [pulumi/pulumi-kubernetes#187]
I've been reading the source to see how it waits for a Kubernetes deployment. Unless I'm missing something, I came across what seems to be a "copy-paste" typo at [pkg/await/apps_deployment.go#L227](https://github.com/pulumi/pulumi-kubernetes/blob/master/pkg/await/apps_deployment.go#L227):

```go
dia.processReplicaSetEvent(watchAddedEvent(pod.(*unstructured.Unstructured)))
```
should be:
```go
dia.processPodEvent(watchAddedEvent(pod.(*unstructured.Unstructured)))
```
🤔

**Comment by hausdorff** [comment]
🔥 😅 🔥 

`Pod`s only provide diagnostic information, so this should have no impact on correctness, but it sure is embarrassing! Will fix ASAP.

**Comment by hausdorff** [comment]
@gitfool Ok this should be fixed in the PR #188.

BTW, happy to answer questions about how the deployment awaiter logic works... alex@pulumi.com, or hit me up in Slack.
